homeassistant/components/backup/manager.py

```diff
@@ -106,11 +106,19 @@ class ManagerBackup(BaseBackup):
     with_automatic_settings: bool | None
 
 
+@dataclass(frozen=True, kw_only=True, slots=True)
+class AddonError:
```

Comment:
The name is a bit confusing considering it's not an exception.

Reply:
Renamed to `AddonErrorData`